src/app/page.tsx

```diff
@@ -255,8 +255,9 @@ async function getPageData(spoofDate?: string) {
 
       const active = drops.filter((drop) => {
         const comparison = compareAsc(now, drop.endDate)
+        const isInfo = drop.address === '0x0' || drop.address === '0x'
```

Comment:
This looks right, but I think we also have to do a quick sanity check against all the drop configs because some of the _actual_ drops (externally linked ones) don't have a contract address. e.g. Showtime. Mind doing that and including in this PR?

Reply:
Oh good point